tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+envlist = py{27,36}-cov
+
+[pytest]
+norecursedirs = data
+
+[testenv]
+changedir = tests
+commands =
+  nocov: pytest -v {posargs}
+  cov: pytest --cov -v {posargs}
+deps =
+  pytest
+  cov: pytest-cov
+
+[testenv:codecov]
+changedir =
+passenv = *
+deps =
+    coverage
+    codecov
+skip_install = true
+ignore_outcome = true
+commands =
+    cd tests
```

Comment:
Shouldn’t we set `changedir = tests` above instead?

Reply:
The upload to Codecov is still not working
```
codecov runtests: commands[0] | cd tests
WARNING:command failed but result from testenv is ignored
  cmd: InvocationError for command could not find executable 'cd'
codecov runtests: commands[1] | coverage combine
No data to combine
WARNING:command failed but result from testenv is ignored
  cmd: InvocationError for command '/home/travis/build/adobe-type-tools/psautohint/.tox/codecov/bin/coverage combine' (exited with code 1)
codecov runtests: commands[2] | codecov --env TOXENV
```

Reply:
I’m not that familiar either, but I think you can’t freely use Unix commands here you need to whitelist them somehow. Let me try my suggestion.